docs/guide/features.md

```diff
@@ -368,6 +370,38 @@ By default, the worker script will be emitted as a separate chunk in the product
 import MyWorker from './worker?worker&inline'
 ```
 
+### new Worker()
```

Comment:
```suggestion
### `new Worker()` and `new SharedWorker()`
```

Reply:
It looks more concise. I also thought of a name. I don't know whether it's good or not. :p

`es suffixes` / `native`

Reply:
Because `the worker suffixes` is used for short below `?worker`. I think it might be better to use abbreviations in the title as soon as possible?

Reply:
Yeah, I've thought about that too and am also not sure what's the right name for it. Maybe `Import with suffixes` / `Import with constructors`

Reply:
how about `import with es query`

Reply:
I don't think we need to mention "es", but query would be a good alternative too. Maybe "Import with query suffixes"? But I'm also fine with just "suffixes".

Reply:
great word